src/Compilers/CSharp/Portable/Binder/WithCrefTypeParametersBinder.cs

```diff
@@ -63,6 +63,7 @@ private MultiDictionary<string, TypeParameterSymbol> CreateTypeParameterMap()
                 case SyntaxKind.IndexerMemberCref:
                 case SyntaxKind.OperatorMemberCref:
                 case SyntaxKind.ConversionOperatorMemberCref:
+                case SyntaxKind.ExtensionMemberCref:
```

Comment:
>case SyntaxKind.ExtensionMemberCref: [](http://example.com/codeflow?start=16&length=36)

Is there a test covering method type parameters for this case?

Reply:
Added `Cref_66`